client/my-sites/people/invite-people/index.jsx

```diff
@@ -361,7 +359,8 @@ const InvitePeople = React.createClass( {
 									disabled={ this.state.sendingInvites } />
 								<FormSettingExplanation>
 									{ this.translate(
-										'(Optional) You can enter a custom message of up to 500 characters that will be included in the invitation to the user(s).'
+										'(Optional) You can enter a custom message of up to 500 characters ' +
```

Comment:
I don't think there will be side effects in this case, but in general I think you're right - considering this it's probably not worth using template literals here, as side effect might come at some point.